pkgs/applications/audio/sfizz/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub , cmake, libjack2, libsndfile }:
+
+stdenv.mkDerivation rec {
+  pname = "sfizz";
+  version = "unstable-2020-01-24";
+
+  src = fetchFromGitHub {
+    owner = "sfztools";
+    repo = pname;
+    rev = "4b116e8cb9844df184d5cf947d6a0fcd9fd3b071";
+    sha256 = "17sy1mp82mg627gkhxalxay9nyslgk15g1146pg28ia1wzyp902z";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ libjack2 libsndfile ];
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DSFIZZ_TESTS=ON"
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/sfztools/sfizz";
+    description = "SFZ jack client and LV2 plugin";
+    license = licenses.bsd2;
+    maintainers = [ maintainers.magnetophon ];
+    platforms = platforms.all;
```

looks like clang doesn't like this
```suggestion
    badPlatforms = platforms.darwin;
```
or
```
  broken = stdenv.cc.isClang;
```

---

OK, done.
Also updated rev again. :)

---

OK, done.
Also updated rev again. :)